src/app/config.ts

```diff
@@ -34,8 +35,23 @@ const config = convict({
 			env: 'MONGODB_URL',
 		},
 	},
+	RECAPTCHA_SECRET_KEY: {
```

Comment:
These config keys should follow the common pattern:
* lowercase keys
* hierarchical structure

So it would look like
```
recaptcha: {
    key: {
        secret: {
            ....
        },
        public: {
            ....
        },
    },
}
```

The language setting can be omitted and rather be hardcoded currently, with a _TODO_ tag that reminds us to inject the correct language once this value comes directly from the system settings.